packages/calcite-components/src/utils/dynamicClasses.ts

```diff
@@ -0,0 +1,3 @@
+export function getDimension(type: "width" | "height", size: string, scale: string): string {
```

Comment:
Can you add a more specific return type? TS supports [template literal types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html).